src/main/kotlin/io/openfuture/state/repository/ProcessingRedisRepository.kt

```diff
@@ -0,0 +1,68 @@
+package io.openfuture.state.repository
+
+import io.openfuture.state.blockchain.Blockchain
+import io.openfuture.state.property.WatcherProperties
+import org.springframework.data.redis.core.*
+import org.springframework.stereotype.Repository
+import java.time.Duration
+import java.time.LocalDateTime
+
+@Repository
+class ProcessingRedisRepository(
+        reactiveRedisTemplate: ReactiveRedisTemplate<String, Any>,
```

Comment:
The type is `ReactiveRedisTemplate`, but the variable could be just `redisTemplate`